### PR TITLE
Pages cleanup

### PR DESCRIPTION
**Pull request by cawel**
New stuff in this PR:

* Removed useless code + useless files from Rails' scaffold
* Added basic validations in the `Page` model
* Ensured a requested page is 'published' (otherwise it should return a 404)
* Added controller and model specs

Note that this PR depends on PR #77 (which is the reason why this PR has a merge commit).

Tagging @kiramclean in case she still wants to be kept in the loop.

### CODE REVIEW

File: spec/models/page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Location, :type => :model do
```
Comment (nicholasjhenry) [inline spec/models/page_spec.rb]: @cawel This should be `Page` not `Location`. :lollipop:

File: Gemfile [hunk 2]
```diff
@@ -27,6 +27,7 @@ gem 'turbolinks'
 
 # View
 gem 'simple_form'
+gem 'enumerize'
```
Comment (nicholasjhenry) [inline Gemfile]: `enumerize` is already included in the Gemfile

https://github.com/montrealrb/Montreal.rb/blob/master/Gemfile#L23

File: db/migrate/20151125004633_create_pages.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreatePages < ActiveRecord::Migration
```
Comment (nicholasjhenry) [inline db/migrate/20151125004633_create_pages.rb]: @cawel Would you be up to make title and body "globalized"? If not happy to accept the PR as is.

https://github.com/globalize/globalize#creating-translation-tables

File: Gemfile (hunk 2 above)
Comment (cawel) [inline Gemfile]: Fixed the offending commit using a rebase interactive.

File: db/migrate/20151125004633_create_pages.rb (hunk 3 above)
Comment (cawel) [inline db/migrate/20151125004633_create_pages.rb]: Fixed the offending commit using a rebase interactive. The `db/schema.rb` should be up to date.

File: spec/models/page_spec.rb (hunk 1 above)
Comment (cawel) [inline spec/models/page_spec.rb]: Fixed the offending commit using a rebase interactive.

**Comment by cawel** [comment]
@nicholasjhenry All fixes done, sir! :rocket:

**Comment by benichu** [comment]
@nicholasjhenry what's the status on that PR? Can we merge it?

File: Gemfile (hunk 2 above)
Comment (nicholasjhenry) [inline Gemfile]: @cawel Unfortunately `enumerize` is still included in this pull-request. In master it's located here:

https://github.com/montrealrb/Montreal.rb/blob/master/Gemfile#L23

**Comment by nicholasjhenry** [comment]
This is good to merge once the following issue has been resolved:

https://github.com/montrealrb/Montreal.rb/pull/79/files#r46279753

File: Gemfile (hunk 2 above)
Comment (cawel) [inline Gemfile]: @nicholasjhenry My understanding of the issue with the `enumerize` gem was that it was _duplicated_ in the Gemfile. So I fixed it a few days ago ([here is the relevant commit](https://github.com/cawel/Montreal.rb/commit/ac85398e4157ca02d14a0c4ad528c7130fdbf722])), changing the Gemfile to include it only once.

Both my `migrations-cleanup` branch and my `pages-cleanup` branch have that gem only once in the Gemfile. I'm not sure about what you expect from me. Is there something I misunderstood?

Comment (nicholasjhenry) [inline Gemfile]: @cawel I'm understand your confusion. I thought this had been rebased with `master` recently. I'm not sure the last time you rebased, but when we merged PR the `enumerize` gem will appear twice in the Gemfile. Please view below:

1. https://github.com/cawel/Montreal.rb/blob/pages-cleanup/Gemfile#L29
2. https://github.com/montrealrb/Montreal.rb/blob/master/Gemfile#L23

My advice is to rebase again, please. Then the Gemfile in your branch should not include an entry for `enumerize`.

Comment (cawel) [inline Gemfile]: I rebased both of my branches on top of `master`. And that Gemfile looks the same to me :)

Comment (nicholasjhenry) [inline Gemfile]: So apparently I'm an idiot (or just my multitasking skills are failing me). As @cawel pointed out the `enumerize` gem was moved, not added -- I missed the deletion for some reason.

Comment (cawel) [inline Gemfile]: No worries @nicholasjhenry ! 

Look 'ma, it's merged! :tada:

**Pull request merged**